build-system/pr-check/build-targets.js

```diff
@@ -202,9 +202,8 @@ function areValidBuildTargets(buildTargets, fileName) {
     console.log(fileLogPrefix, colors.red('ERROR:'),
         'Please move these files to a separate PR:',
         colors.cyan(nonFlagConfigFiles.join(', ')));
-    return false;
+    process.exit(1);
```

Comment:
Is there a reason for this change? Node docs recommend against using `process.exit()` directly [(reasons here)](https://nodejs.org/api/process.html#process_process_exit_code)

Reply:
Also I like keeping methods as simple as possible; killing a Node process in `validateBuildTargets()` makes assumptions about the caller and kind of oversteps what the function is supposed to do per the name, which is validate.